packages/system/robotlb/Makefile

```
@@ -0,0 +1,9 @@
+export NAME=hcloud-ccm
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)